app/assets/stylesheets/barnardos/_type.scss

```diff
@@ -79,24 +77,24 @@ p.short-text {
 
   font-size: 16px;
   line-height: 16px;
-  margin: 0 0 24px;
+  margin: 24px 0;
 
   @media (min-width: $tablet) {
     font-size: 18px;
     line-height: 24px;
-    margin: 0 0 32px;
+    margin: 32px 0;
   }
 }
 
 .super-text,
 p.super-text {
   font-size: 18px;
   line-height: 24px;
-  margin: 0 0 24px;
+  margin: 24px 0;
 
   @media (min-width: $tablet) {
     font-size: 23px;
     line-height: 36px;
-    margin: 0 0 32px;
+    margin: 32px 0;
```

Comment:
We use multiples of 8 and 16. It may be worth having a separate PR to put these in as named units